### ISSUE TITLE
Rule not wrapped when the first selector starts with the prefix

### ISSUE THREAD

**Issue by mevtho** [dbtedman/postcss-prefixwrap#146]
Hi,

I run into an issue when trying to wrap rules where the first selector starts with the prefix.

Example :

There are issue with rules starting with `.test-4` (`.test-4`, `.test-4, .test-5`, `.test-4, .other`) selectors in the example below:

Trying to wrap the following with the prefix `.te`:

```css
p {
    color: #ff0000;
}

.test-4 {
    color: #ff0000;
}

.test-4, .test-5 {
    color: #ff0000;
}

.test-4, .other {
    color: #ff0000;
}

h1.test-4, .test-5 {
    color: #ff0000;
}

.not-test-4 {
    color: #ff0000;
}

.other {
    color: #ff0000;
}
```

results in:

```css
.te p {
    color: #ff0000;
}

.test-4 {
    color: #ff0000;
}

.test-4, .test-5 {
    color: #ff0000;
}

.test-4, .other {
    color: #ff0000;
}

.te h1.test-4, .te .test-5 {
    color: #ff0000;
}

.te .not-test-4 {
    color: #ff0000;
}

.te .other {
    color: #ff0000;
}
```

when the expected output would be:

```css
.te p {
    color: #ff0000;
}

.te .test-4 {
    color: #ff0000;
}

.te .test-4, .te .test-5 {
    color: #ff0000;
}

.te .test-4, .te .other {
    color: #ff0000;
}

.te h1.test-4, .te .test-5 {
    color: #ff0000;
}

.te .not-test-4 {
    color: #ff0000;
}

.te .other {
    color: #ff0000;
}
```

Not necessarily a big issue as it is easy enough to change / add a letter to make it different, but it was enough to trip me up when trying to wrap blueprintjs's css with `.bp` (most rules starting with `.bp3-`). Examples extracted from my observations in that case.

Note: I can try to fix it sometime within the next week or so unless anyone does before.

**Comment by dbtedman** [comment]
Hi @mevtho thanks for reporting this issue, I appreciate the explanation and examples, I will take a look to see what I can do to resolve this issue.

**Comment by dbtedman** [comment]
A fix has been prepared, and I have refactored how the code functions for detecting if a selector is the prefix selector instead of just containing the selector. A new release containing this update will be completed soon.

**Comment by dbtedman** [comment]
@mevtho this has been released as version 1.28.1

**Comment by mevtho** [comment]
@dbtedman Awesome! Thank you very much. I just tried on my initial use case and it seems to be working as expected.
